From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 20 Feb 2021 19:58:08 -0800
Subject: libevdev: update to 1.11.0

Use AUTORELEASE to avoid having to bump it manually.

Remove old patch for kernel 4.14 compatibility.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

diff --git a/libs/libevdev/Makefile b/libs/libevdev/Makefile
index fcc9da1911223dcdc8db165222fc23696a0f2b67..31b9213707359cf3b1e3a774041e1212ea058f75 100644
--- a/libs/libevdev/Makefile
+++ b/libs/libevdev/Makefile
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libevdev
-PKG_VERSION:=1.10.1
-PKG_RELEASE:=1
+PKG_VERSION:=1.11.0
+PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.freedesktop.org/software/libevdev/
-PKG_HASH:=0330fe8357ece915db9366c1b9a6648941aea6f724b73ad6e71401127aa08932
+PKG_HASH:=63f4ea1489858a109080e0b40bd43e4e0903a1e12ea888d581db8c495747c2d0
 
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
 PKG_LICENSE:=MIT
diff --git a/libs/libevdev/patches/010-libevdev-events-fix-compilation-with-older-kernels.patch b/libs/libevdev/patches/010-libevdev-events-fix-compilation-with-older-kernels.patch
deleted file mode 100644
index 031911c45aa6a19432dcab636650af08ddfa3c61..0000000000000000000000000000000000000000
--- a/libs/libevdev/patches/010-libevdev-events-fix-compilation-with-older-kernels.patch
+++ /dev/null
@@ -1,42 +0,0 @@
-From e7a2927d5e44209ff61405c9beb3d188eeedc3f6 Mon Sep 17 00:00:00 2001
-From: Rosen Penev <rosenp@gmail.com>
-Date: Sun, 26 Apr 2020 20:27:58 -0700
-Subject: [PATCH] libevdev-events: fix compilation with older kernels
-
-input_event_(u)sec was introduced in kernel 4.16. Older kernels do not
-have this.
-
-Signed-off-by: Rosen Penev <rosenp@gmail.com>
----
- tools/libevdev-events.c | 5 +++++
- tools/mouse-dpi-tool.c  | 5 +++++
- 2 files changed, 10 insertions(+)
-
---- a/tools/libevdev-events.c
-+++ b/tools/libevdev-events.c
-@@ -33,6 +33,11 @@
- 
- #include "libevdev/libevdev.h"
- 
-+#ifndef input_event_sec
-+#define input_event_sec time.tv_sec
-+#define input_event_usec time.tv_usec
-+#endif
-+
- static void
- print_abs_bits(struct libevdev *dev, int axis)
- {
---- a/tools/mouse-dpi-tool.c
-+++ b/tools/mouse-dpi-tool.c
-@@ -37,6 +37,11 @@
- 
- #include "libevdev/libevdev.h"
- 
-+#ifndef input_event_sec
-+#define input_event_sec time.tv_sec
-+#define input_event_usec time.tv_usec
-+#endif
-+
- #define min(a, b) (((a) < (b)) ? (a) : (b))
- #define max(a, b) (((a) > (b)) ? (a) : (b))
- 
